rskj-core/src/main/java/co/rsk/peg/pegin/PeginProcessAction.java

```diff
@@ -0,0 +1,7 @@
+package co.rsk.peg.pegin;
+
+public enum PeginProcessAction {
+    CAN_BE_REGISTERED,
+    CAN_BE_REFUNDED,
+    CANNOT_BE_PROCESSED
```

Comment:
Being nit picky here, but the names `CAN_BE` don't convince me. I feel it should be something more direct, like SHOULD_REFUND. For example

```suggestion
    REGISTER,
    REFUND,
    UNPROCESSABLE
```
I'm not convinced about the last one UNPROCESSABLE isn't really an action. Maybe `IGNORE`?

Reply:
Well, the enum is called PeginProcessAction so one might expect it to contain actions that are in fact verbs right?

Maybe use a different name for the enum?